src/helm/benchmark/metrics/classification_metrics.py

```diff
@@ -68,3 +70,38 @@ def evaluate_instances(self, request_states: List[RequestState]) -> List[Stat]:
             Stat(MetricName("classification_macro_f1")).add(f1_score(y_pred=y_pred, y_true=y_true, average="macro")),
             Stat(MetricName("classification_micro_f1")).add(f1_score(y_pred=y_pred, y_true=y_true, average="micro")),
         ]
+
+
+class CLEVAMultipleChoiceClassificationMetric(Metric):
```

Comment:
Yes, you are right.

Reply:
Sounds good. I think you can just drop `CLEVA` from the name since this can be used for the non-CLEVA scenarios as well.

Likewise for "CLEVA" in the remainder of the comments and function body.

Reply:
Thanks for your suggestions! We drop `CLEVA` in the latest commits.